components/bio-formats-tools/test/loci/formats/tools/ImageConverterTest.java

```diff
@@ -124,6 +133,25 @@ public void testDefault(String suffix) throws FormatException, IOException {
     }
   }
 
+  @Test(dataProvider = "options")
+  public void testOptions(String options) throws FormatException, IOException {
+    outFile = File.createTempFile("test", ".ome.tiff");
+    outFile.delete();
```

Comment:
Ahhh ok that makes sense.! Would be great if we could add a line of comment in such cases. It would help other developers to understand the reason behind this implementation as well (given that this is not intuitive while reading through the code). Happy to be corrected as well,just a suggestion.